src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/DampingFunction.hpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Parallel/CharmPupable.hpp"
+
+class DataVector;
```

Comment:
Need to `/// \cond` the forward decl

Reply:
Sorry I missed those comments and had so many include mistakes! I'll try to catch these things next time

Reply:
Nothing to apologize for! I appreciate you squeezing in the time to update the code :)